malariagen_data/anoph/karyotype_params.py

```diff
@@ -0,0 +1,10 @@
+"""Parameter definitions for karyotype analysis functions."""
+
+from typing import Literal
+
+from typing_extensions import Annotated, TypeAlias
+
+inversion_param: TypeAlias = Annotated[
+    Literal["2La", "2Rb", "2Rc_gam", "2Rc_col", "2Rd", "2Rj"],
+    "Name of inversion to infer karyotype for.",
+]
```

Comment:
These values will only be valid for Ag. Af would have different possible values. Suggest broadening here to just a string.
```suggestion
inversion_param: TypeAlias = Annotated[
    str,
    "Name of inversion to infer karyotype for.",
]
```